src/tcp-client-test/LineTcpSenderTests.cs

```diff
@@ -34,7 +34,8 @@
 using Org.BouncyCastle.Crypto.Parameters;
 using Org.BouncyCastle.Math;
 using Org.BouncyCastle.Security;
-using QuestDB;
+using QuestDB.Ingress;
+using QuestDB.Ingress.Enums;
```

Comment:
New usings should not be necessary, legacy code should compile without additional namespaces.